src/shared_modules/keystore/include/keyStore.hpp

```diff
@@ -88,6 +92,7 @@ class TKeystore final
             }
             // Decrypt value
             TRSAPrimitive().rsaDecrypt(PRIVATE_KEY_FILE, encryptedValue, value);
+            logDebug2(KS_NAME, "Decryption successful for key: '%s'", key.c_str());
```

Comment:
We can keep this log because it is debug and by default it does not appear in the ossec.log